doc/playbook.rst

```diff
@@ -558,6 +558,27 @@ To restore the APDB to a clean state, add the ``--drop`` option to  ``apdb-cli c
    apdb-cli create-sql --drop --namespace="pp_apdb_lsstcomcamsim" \
        "postgresql://rubin@usdf-prompt-processing-dev.slac.stanford.edu/lsst-devl" apdb_config_lsstcomcamsim.py
 
+Removing APDB Instances
+-----------------------
```

Comment:
I realize I'm butting in, but what exactly is the motivation here? In the dev environment we should only have one APDB per instrument, and while we may want to _reset_ it, we almost never want to just outright delete it -- doing so would break the dev service(s).

The only time we've tried to delete an APDB, permanently, were the ones created for sanity-checking the 1.0.0 and 1.1.0 migrations, and the versioning system seems to be a little more stable now.

Reply:
My guess is that people will create multiple APDB instances for various tests. Postgres `usdf-prompt-processing-dev/lsst-devl` contains more than 100 schemas/namespaces for APDB instances. If people can create those instances freely, it may be worth adding an ability to drop them as well?

Reply:
Those namespaces are from the precursor processing group -- completely different workflow, and they don't (and shouldn't) read these docs.

Reply:
So, should I drop all of that? I'm OK with whatever you propose, the help for `apdb-cli` should be sufficient in case someone needs it.

Reply:
Yes, let's drop this from the playbook documentation. The proper place would be in https://pipelines.lsst.io , and I will make a ticket to update the documentation there with the text you wrote for this ticket.

Reply:
OK, closing this PR.